Formula/kmod.rb

```diff
@@ -0,0 +1,36 @@
+class Kmod < Formula
+  desc "Linux kernel module handling"
+  homepage "https://git.kernel.org/?p=utils/kernel/kmod/kmod.git"
+  url "https://www.kernel.org/pub/linux/utils/kernel/kmod/kmod-29.tar.xz"
+  sha256 "0b80eea7aa184ac6fd20cafa2a1fdf290ffecc70869a797079e2cc5c6225a52a"
+  license all_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later"]
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gcc" => :build
+  depends_on "gtk-doc" => :build
+  depends_on "libtool" => :build
+  depends_on "m4" => :build
+  depends_on "make" => :build
+  depends_on :linux
+
+  def install
+    system "autoreconf", "-fiv"
+    system "./configure", "--with-bashcompletiondir=#{bash_completion}",
+      *std_configure_args, "--disable-silent-rules"
+    system "make"
+    system "make", "install"
+
+    bin.install_symlink "kmod" => "depmod"
+    bin.install_symlink "kmod" => "lsmod"
+    bin.install_symlink "kmod" => "modinfo"
+    bin.install_symlink "kmod" => "insmod"
+    bin.install_symlink "kmod" => "modprobe"
+    bin.install_symlink "kmod" => "rmmod"
```

Comment:
Does `make install` not do this?

Reply:
Unfortunately doesn't seem to install them.

Reply:
Could you suggest it upstream?

Reply:
To be honest, I don't have a strong interest in getting involved with upstream there (my desire to patch an autotools build system is also not that great).